src/presentation/wallet/transactions/index.tsx

```diff
@@ -67,11 +67,6 @@ const TransactionsView: React.FC<TransactionsProps> = ({
     await browser.tabs.create({ url, active: false });
   };
 
-  // Update txs history once at first render
```

Comment:
how do we handle this now?

Reply:
I've reverted the effect in order to dispatch an update task for the MainAccount. Later, if we implement multiple accounts feature we could maybe improve this but now it's ok imo